package.json

```diff
@@ -1,18 +1,19 @@
 {
   "packageManager": "pnpm@7.27.1",
-  "name": "turbo",
+  "name": "webstudio",
   "version": "0.0.0",
   "private": true,
   "scripts": {
     "": "For some reason remix build does not finish when run with turbo or pnpm recursively",
-    "build-packages": "pnpm run --recursive --parallel --filter='!./apps/*' --filter='!./packages/sdk-size-test' build",
-    "build": "pnpm build-packages && pnpm --filter=@webstudio-is/sdk-size-test build && pnpm --filter=@webstudio-is/builder build",
+    "build": "turbo run build",
+    "build-packages": "turbo run build --filter=\"!@webstudio-is/builder\"",
+    "size-test": "turbo run size-test",
     "checks": "turbo run checks",
     "dev": "turbo run dev --parallel",
     "format": "prettier --write \"**/*.{ts,tsx,md}\"",
     "build-storybook": "build-storybook",
     "storybook": "turbo run storybook:run",
-    "storybook:build": "npm run build && turbo run storybook:build",
+    "storybook:build": "turbo run storybook:build",
```

Comment:
Previously this ran a build, and then ran storybook build, duplicating the effort. By pushing all of this into Turborepo's execution graph there is no longer any duplicated work.